### PR TITLE
New design for integrate to django-netjsonconfig

### PR DESCRIPTION
**Pull request by mapeveri**
image

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "netjsonconfig-editor.js",
-  "version": "1.0.0",
+  "version": "1.0.1",
```
Comment (nemesisdesign) [inline package-lock.json]: let's change this to 0.1.0

File: package-lock.json [hunk 2]
```diff
@@ -3184,905 +3183,6 @@
...
-            "string-width": "1.0.2"
-          }
-        },
-        "wrappy": {
-          "version": "1.0.2",
-          "bundled": true,
-          "dev": true
-        }
-      }
-    },
```
Comment (nemesisdesign) [inline package-lock.json]: why all these removed? Was this redundant or what?

File: package-lock.json [hunk 3]
```diff
@@ -5749,13 +4849,6 @@
       "integrity": "sha1-MHXOk7whuPq0PhvE2n6BFe0ee6s=",
       "dev": true
     },
-    "nan": {
-      "version": "2.7.0",
-      "resolved": "https://registry.npmjs.org/nan/-/nan-2.7.0.tgz",
-      "integrity": "sha1-2Vv3IeyHfgjbJ27T/G63j5CDrUY=",
-      "dev": true,
-      "optional": true
-    },
```
Comment (nemesisdesign) [inline package-lock.json]: why this is removed? Was this redundant or what?

File: src/js/basic/app.js [hunk 4]
```diff
@@ -69,25 +69,18 @@ class App extends React.Component {
...
-      className="advanced-button btn btn-block"
-      onClick={() => {
-        props.swapOut();
-      }}
-    >
-      Advanced Mode
-    </button>
-  </div>
-</div>);
+);
```
Comment (nemesisdesign) [inline src/js/basic/app.js]: 👍  seems more readable

File: package-lock.json (hunk 1 above)
Comment (mapeveri) [reply package-lock.json]: Ready :)

File: package-lock.json (hunk 2 above)
Comment (mapeveri) [reply package-lock.json]: This file is generated by npm.

File: package-lock.json (hunk 3 above)
Comment (mapeveri) [reply package-lock.json]: This file is generated by npm

File: src/js/basic/app.js (hunk 4 above)
Comment (mapeveri) [reply src/js/basic/app.js]: Perfect :+1:

**Comment by nemesisdesign** [comment]
@mapeveri thanks :+1: !

**Pull request merged**